src-tool/Pact/Analyze/Types/Languages.hs

```diff
@@ -1654,7 +1654,7 @@ prettyTerm ty = \case
   MkPactGuard name -> parensSep ["create-pact-guard", pretty name]
   MkUserGuard g t -> parensSep ["create-user-guard", pretty g, pretty t]
   MkModuleGuard name -> parensSep ["create-module-guard", pretty name]
-  MkCapabilityGuard (Capability _ n) as isPact -> parensSep
+  MkCapabilityGuard (Capability _ n _) as isPact -> parensSep
```

Comment:
Does `CapAnnot` not want to be included in the pretty-printed `Capability`?

Reply:
I think a cap being managed should display somehow. Event is something that will someday be replaced by `defevent`, I believe.